apps/modernization-ui/src/components/Table/Table.tsx

```diff
@@ -55,7 +55,7 @@ export type Props = {
     selectable?: boolean;
     handleSelected?: OldSelectionHandler;
     isLoading?: boolean;
-    contextName?: 'pages' | 'conditions' | 'questions' | 'valuesets' | 'templates';
+    context?: any;
```

Comment:
Is there a reason why we are passing context through props?

We should use createContext and useContext and not need to pass through props:
https://react.dev/learn/passing-data-deeply-with-context

Reply:
Since the table is shared by multiple pages, i was passing the context name so that the table knew which context to use.  Not sure why it was changed here to context?: any

Reply:
This change will also break the functionality added to the <RangeToggle> component for page size/page number.